src/qt/fontenum.cpp

```diff
@@ -7,13 +7,23 @@
 
 // For compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
-
 #include "wx/fontenum.h"
+#include <QFontDatabase>
 
-bool wxFontEnumerator::EnumerateFacenames( wxFontEncoding WXUNUSED(encoding),
-                bool WXUNUSED(fixedWidthOnly))
+bool wxFontEnumerator::EnumerateFacenames( wxFontEncoding WXUNUSED(encoding), bool fixedWidthOnly)
 {
-    return false;
+    QFontDatabase fontDatabase;
+    const QStringList allFonts = fontDatabase.families(QFontDatabase::Any);
```

Comment:
```suggestion
    const QStringList& allFonts = fontDatabase.families(QFontDatabase::Any);
```

Reply:
QFontDatabase::families returns a value not a reference.  We don't indent to modify this I've made it immutable.

Reply:
> QFontDatabase::families returns a value not a reference.

Right, and you don't need to make a copy of it, just keep a const reference to it.